std_msgs/Header header
string class_name
float32 score
int32[] contour_x
int32[] contour_y
float32[4] bbox
